doc/mgmt/OpenConfig_PortChannel_Interface.md

```diff
@@ -0,0 +1,279 @@
+        |     |        +--rw oc-ip:config
+        |     |        |  +--rw oc-ip:ip?              oc-inet:ipv4-address
+        |     |        |  +--rw oc-ip:prefix-length?   uint8
+        |     |        +--ro oc-ip:state
+        |     |           +--ro oc-ip:ip?              oc-inet:ipv4-address
+        |     |           +--ro oc-ip:prefix-length?   uint8
+        |     +--rw oc-ip:ipv6
+        |        +--rw oc-ip:addresses
+        |        |  +--rw oc-ip:address* [ip]
+        |        |     +--rw oc-ip:ip        -> ../config/ip
+        |        |     +--rw oc-ip:config
+        |        |     |  +--rw oc-ip:ip?              oc-inet:ipv6-address
+        |        |     |  +--rw oc-ip:prefix-length    uint8
+        |        |     +--ro oc-ip:state
+        |        |        +--ro oc-ip:ip?              oc-inet:ipv6-address
+        |        |        +--ro oc-ip:prefix-length    uint8
+        |        +--rw oc-ip:config
+        |        |  +--rw oc-ip:enabled?   boolean
+        |        +--ro oc-ip:state
+        |           +--ro oc-ip:enabled?   boolean
+        +--rw oc-eth:ethernet
+        |  +--rw oc-eth:config
+        |  |  +--rw oc-eth:auto-negotiate?   boolean
+        |  |  +--rw oc-eth:port-speed?       identityref
+        |  |  +--rw oc-lag:aggregate-id?     -> /oc-if:interfaces/interface/name
+        |  +--ro oc-eth:state
+        |     +--ro oc-eth:auto-negotiate?   boolean
+        |     +--ro oc-eth:port-speed?       identityref
+        |     +--ro oc-eth:counters
+        |     |  +--ro oc-eth:in-oversize-frames?    oc-yang:counter64
+        |     |  +--ro oc-eth:in-undersize-frames?   oc-yang:counter64
+        |     |  +--ro oc-eth:in-jabber-frames?      oc-yang:counter64
+        |     |  +--ro oc-eth:in-fragment-frames?    oc-yang:counter64
+        |     +--ro oc-lag:aggregate-id?     -> /oc-if:interfaces/interface/name
+        +--rw oc-lag:aggregation
+           +--rw oc-lag:config
+           |  +--rw oc-lag:min-links?   uint16
+           +--ro oc-lag:state
+              +--ro oc-lag:min-links?   uint16
+```
+# Definition/Abbreviation
+### Table 1: Abbreviations
+| **Term**                 | **Definition**                         |
+|--------------------------|-------------------------------------|
+| YANG                     | Yet Another Next Generation: modular language representing data structures in an XML tree format        |
+| REST | REpresentative State Transfer |
+| gNMI                     | gRPC Network Management Interface: used to retrieve or manipulate the state of a device via telemetry or configuration data         |
+| XML                     | eXtensible Markup Language   |
+| Aggregate |   Interchangable with portchannel |
+
+
+# 1 Feature Overview
+## 1.1 Requirements
+### 1.1.1 Functional Requirements
+1. Provide support for OpenConfig YANG models.
+2. Replace translib App based implementation with a Transformer based implementation of:
```

Comment:
Is there translib support for portchannel? Otherwise we can remove this line

Reply:
You are right, it seems it was not supported before. Will remove the line, thank you! 
Will mark as resolved after pushing fixes.